src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -654,11 +738,44 @@ export default {
 }
 
 .conversations-search {
-	flex-grow: 1;
-
+	transition: all 0.3s ease;
+	z-index: 1;
+	// new conversation button width : 52 px
+	// Filters button width : 44 px
```

Comment:
Could be replaced with `var(--default-clickable-area)`

+ to Joas' commit: we could decrease width up to 3 more px to get a nice gap between input and button